spec/passaporte_web/service_account_spec.rb

```diff
@@ -130,32 +130,32 @@
     let(:service_account) { PassaporteWeb::ServiceAccount.find("859d3542-84d6-4909-b1bd-4f43c1312065") }
     context "on success" do
       it "should update the ServiceAccount attributes on the server" do
-        service_account.plan_slug.should == 'free'
-        service_account.expiration.should == '2014-04-01 00:00:00'
+        expect(service_account.plan_slug).to eq('free')
+        expect(service_account.expiration).to eq('2014-04-01 00:00:00')
 
         service_account.plan_slug = 'basic'
         service_account.expiration = '2014-05-01'
 
-        service_account.should be_persisted
+        expect(service_account).to be_persisted
         expect(service_account.save).to be_truthy
-        service_account.should be_persisted
+        expect(service_account).to be_persisted
 
-        service_account.plan_slug.should == 'basic'
-        service_account.expiration.should == '2014-05-01 00:00:00'
+        expect(service_account.plan_slug).to eq('basic')
+        expect(service_account.expiration).to eq('2014-05-01 00:00:00')
 
         service_account = PassaporteWeb::ServiceAccount.find("859d3542-84d6-4909-b1bd-4f43c1312065")
-        service_account.plan_slug.should == 'basic'
-        service_account.expiration.should == '2014-05-01 00:00:00'
+        expect(service_account.plan_slug).to eq('basic')
+        expect(service_account.expiration).to eq('2014-05-01 00:00:00')
       end
     end
     context "on failure" do
       it "should return false and set the errors hash" do
         service_account.plan_slug = nil # required
         service_account.expiration = nil
-        service_account.should be_persisted
+        expect(service_account).to be_persisted
         expect(service_account.save).to be_falsy
-        service_account.should be_persisted
-        service_account.errors.should == {"field_errors"=>{"plan_slug"=>["Este campo é obrigatório."]}}
+        expect(service_account).to be_persisted
+        expect(service_account.errors).to eq({"field_errors"=>{"plan_slug"=>["Este campo é obrigatório."]}})
```

Comment:
Redundant curly braces around a hash parameter.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.